activiti-core/activiti-engine/src/test/java/org/activiti/engine/test/api/runtime/ProcessInstanceQueryTest.java

```diff
@@ -580,6 +582,31 @@ public void testQueryInvalidSorting() {
       .isThrownBy(() -> runtimeService.createProcessInstanceQuery().orderByProcessDefinitionId().list());
   }
 
+  @Deployment(resources = {"org/activiti/engine/test/api/oneTaskProcessWithCandidateUsersAndGroups.bpmn20.xml"})
```

Comment:
isn't `testQueryCandidateOrAssignedUser` enough for testing candidateOrAssingee and canididates ?